msgid ""
msgstr ""
"Project-Id-Version: pyload\n"
"Report-Msgid-Bugs-To: 'bugs@pyload.org'\n"
"POT-Creation-Date: 2014-07-13 20:53+0200\n"
"PO-Revision-Date: 2015-01-18 18:30-0500\n"
"Last-Translator: pyloadTeam <team@pyload.org>\n"
"Language-Team: Spanish\n"
"Language: es_ES\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: crowdin.com\n"
"X-Crowdin-Project: pyload\n"
"X-Crowdin-File: /pyLoad/core.pot\n"

#: pyLoadCore.py:181
msgid "Received Quit signal"
msgstr "Señal de salida recibida"

#: pyLoadCore.py:302
#, python-format
msgid "pyLoad already running with pid %s"
msgstr "pyLoad ya está ejecutándose con pid %s"

#: pyLoadCore.py:316
#, python-format
msgid "Failed changing group: %s"
msgstr "Cambio de grupo fallido: %s"

#: pyLoadCore.py:326
#, python-format
msgid "Failed changing user: %s"
msgstr "Cambio de usuario fallido: %s"

#: pyLoadCore.py:328
msgid "folder for logs"
msgstr "carpeta para los registros"

#: pyLoadCore.py:339
msgid "Starting"
msgstr "Iniciando"

#: pyLoadCore.py:340
#, python-format
msgid "Using home directory: %s"
msgstr "Usando directorio de inicio: %s"

#: pyLoadCore.py:349
msgid "pycrypto to decode container files"
msgstr "pycrypto va a decodificar archivos contenedores"

#: pyLoadCore.py:352
msgid "folder for temporary files"
msgstr "carpeta para archivos temporales"

#: pyLoadCore.py:357
msgid "folder for downloads"
msgstr "carpeta de descargas"

#: pyLoadCore.py:360
msgid "OpenSSL for secure connection"
msgstr "OpenSSL para conexión segura"

#: pyLoadCore.py:364
msgid "Moving old user config to DB"
msgstr "Moviendo la configuración antigua de usuario a la BD"

#: pyLoadCore.py:367
msgid "Please check your logindata with ./pyLoadCore.py -u"
msgstr "Por favor, comprueba tus datos de inicio de sesión con ./pyLoadCore.py -u"

#: pyLoadCore.py:370
msgid "All links removed"
msgstr "Se han eliminado todos los enlaces"

#: pyLoadCore.py:401
#, python-format
msgid "Downloadtime: %s"
msgstr "Tiempo de descarga: %s"

#: pyLoadCore.py:411
#, python-format
msgid "Free space: %s"
msgstr "Espacio libre: %s"

#: pyLoadCore.py:431
msgid "Activating Accounts..."
msgstr "Activando Cuentas..."

#: pyLoadCore.py:437
msgid "Activating Plugins..."
msgstr "Activando Plugins..."

#: pyLoadCore.py:440
msgid "pyLoad is up and running"
msgstr "pyLoad está funcionando"

#: pyLoadCore.py:459
msgid "restarting pyLoad"
msgstr "reiniciando pyLoad"

#: pyLoadCore.py:463
msgid "pyLoad quits"
msgstr "pyLoad se cierra"

#: pyLoadCore.py:520
#, python-format
msgid "Install %s"
msgstr "Instalar %s"

#: pyLoadCore.py:556
#, python-format
msgid "could not find %(desc)s: %(name)s"
msgstr "no se pudo encontrar %(desc)s: %(name)s"

#: pyLoadCore.py:558
#, python-format
msgid "could not create %(desc)s: %(name)s"
msgstr "no se pudo crear %(desc)s: %(name)s"

#: pyLoadCore.py:579
msgid "shutting down..."
msgstr "apagando..."

#: pyLoadCore.py:596
msgid "error while shutting down"
msgstr "ha ocurrido un error mientras se apagaba"

#: pyLoadCore.py:660
msgid "killed pyLoad from Terminal"
msgstr "pyload finalizado desde el Terminal"

#: module/database/DatabaseBackend.py:174
msgid "Filedatabase was deleted due to incompatible version."
msgstr "El archivo de la base de datos fue eliminado debido a una versión incompatible."

#: module/database/DatabaseBackend.py:189
msgid "Filedatabase could NOT be converted."
msgstr "El archivo de la base de datos NO pudo convertirse."

#: module/database/DatabaseBackend.py:198
msgid "Database was converted from v2 to v3."
msgstr "La base de datos se convirtió de la v2 a la v3."

#: module/database/DatabaseBackend.py:206
msgid "Database was converted from v3 to v4."
msgstr "La base de datos se convirtió de la v3 a la v4."

#: module/database/DatabaseBackend.py:252
msgid "Converting old Django DB"
msgstr "Convirtiendo la BD Django antigua"

#: module/database/FileDatabase.py:45
msgid "finished"
msgstr "terminado"

#: module/database/FileDatabase.py:45
msgid "offline"
msgstr "sin conexión"

#: module/database/FileDatabase.py:45
msgid "online"
msgstr "en línea"

#: module/database/FileDatabase.py:45
msgid "queued"
msgstr "en cola"

#: module/database/FileDatabase.py:45
msgid "skipped"
msgstr "omitido"

#: module/database/FileDatabase.py:45
msgid "waiting"
msgstr "esperando"

#: module/database/FileDatabase.py:45
msgid "temp. offline"
msgstr "temp. sin conexión"

#: module/database/FileDatabase.py:45
msgid "starting"
msgstr "comenzando"

#: module/database/FileDatabase.py:45
msgid "failed"
msgstr "fallido"

#: module/database/FileDatabase.py:45
msgid "aborted"
msgstr "abortado"

#: module/database/FileDatabase.py:45
msgid "decrypting"
msgstr "descifrando"

#: module/database/FileDatabase.py:45
msgid "custom"
msgstr "personalizado"

#: module/database/FileDatabase.py:45
msgid "downloading"
msgstr "descargando"

#: module/database/FileDatabase.py:45
msgid "processing"
msgstr "procesando"

#: module/database/FileDatabase.py:45
msgid "unknown"
msgstr "desconocido"

#: module/database/FileDatabase.py:531 module/plugins/hooks/IRCInterface.py:74
#: module/plugins/hooks/XMPPInterface.py:83
#, python-format
msgid "Package finished: %s"
msgstr "Paquete terminado: %s"

#: module/remote/ThriftBackend.py:39
msgid "Using SSL ThriftBackend"
msgstr "Usando SSL ThriftBackend"

#: module/remote/RemoteManager.py:35
#, python-format
msgid "Remote backend error: %s"
msgstr "Error de backend remoto: %s"

#: module/remote/RemoteManager.py:82
#, python-format
msgid "Starting %(name)s: %(addr)s:%(port)s"
msgstr "Iniciando %(name)s: %(addr)s:%(port)s"

#: module/remote/RemoteManager.py:84
#, python-format
msgid "Failed loading backend %(name)s | %(error)s"
msgstr "Fallo al cargar backend %(name)s | %(error)s"

#: module/web/json_app.py:57
#, python-format
msgid "waiting %s"
msgstr "esperando a %s"

#: module/web/ServerThread.py:36
msgid "SSL certificates not found."
msgstr "Certificados SSL no encontrados."

#: module/web/ServerThread.py:40
#, python-format
msgid "Sorry, we dropped support for starting %s directly within pyLoad"
msgstr "Lo sentimos, ya no admitimos iniciar %s directamente en pyLoad"

#: module/web/ServerThread.py:41
msgid "You can use the threaded server which offers good performance and ssl,"
msgstr "Puedes utilizar el servidor con hilos que ofrece un buen rendimiento y ssl,"

#: module/web/ServerThread.py:42
#, python-format
msgid "of course you can still use your existing %s with pyLoads fastcgi server"
msgstr "por supuesto puedes usar aún tu %s existente con el servidor fastcgi de pyLoad"

#: module/web/ServerThread.py:43
msgid "sample configs are located in the module/web/servers directory"
msgstr "las configuraciones de ejemplo se encuentran en el directorio module/web/servers"

#: module/web/ServerThread.py:49
#, python-format
msgid "Can't use %(server)s, python-flup is not installed!"
msgstr "¡No puedes usar %(server)s, python-flup no está instalado!"

#: module/web/ServerThread.py:58
#, python-format
msgid "Error importing lightweight server: %s"
msgstr "Error importando servidor ligero: %s"

#: module/web/ServerThread.py:59
msgid "You need to download and compile bjoern, https://github.com/jonashaag/bjoern"
msgstr "Necesitas descargar y compilar bjoern, https://github.com/jonashaag/bjoern"

#: module/web/ServerThread.py:60
msgid "Copy the boern.so to module/lib folder or use setup.py install"
msgstr "Copia el módulo boern.so a la carpeta module/lib o usa la instalación setup.py"

#: module/web/ServerThread.py:61
msgid "Of course you need to be familiar with linux and know how to compile software"
msgstr "Por supuesto debes estar familiarizado con linux y saber cómo compilar software"

#: module/web/ServerThread.py:64
msgid "Server set to threaded, due to known performance problems on windows."
msgstr "Servidor en modo threaded, debido a problemas conocidos de rendimiento en Windows."

#: module/web/ServerThread.py:80 module/web/ServerThread.py:103
msgid "This server offers no SSL, please consider using threaded instead"
msgstr "Este servidor no ofrece SSL, por favor considera el usar el servidor con hilos en su lugar"

#: module/web/ServerThread.py:82
#, python-format
msgid "Starting builtin webserver: %(host)s:%(port)d"
msgstr "Iniciando servidor web incorporado: %(host)s:%(port)d"

#: module/web/ServerThread.py:87
#, python-format
msgid "Starting threaded SSL webserver: %(host)s:%(port)d"
msgstr "Iniciando servidor web SSL con hilos : %(host)s:%(port)d"

#: module/web/ServerThread.py:91
#, python-format
msgid "Starting threaded webserver: %(host)s:%(port)d"
msgstr "Iniciando servidor web con hilos: %(host)s:%(port)d"

#: module/web/ServerThread.py:97
#, python-format
msgid "Starting fastcgi server: %(host)s:%(port)d"
msgstr "Iniciando servidor fastcgi: %(host)s:%(port)d"

#: module/web/ServerThread.py:105
#, python-format
msgid "Starting lightweight webserver (bjoern): %(host)s:%(port)d"
msgstr "Iniciando servidor web ligero (bjoern): %(host)s:%(port)d"

#: module/web/pyload_app.py:127
msgid "You dont have permission to access this page."
msgstr "No tienes permiso para acceder a esta página."

#: module/web/pyload_app.py:195
msgid "Download directory not found."
msgstr "No se encuentra el directorio de descarga."

#: module/web/pyload_app.py:262 module/web/pyload_app.py:269
msgid "unlimited"
msgstr "ilimitado"

#: module/web/pyload_app.py:264 module/web/pyload_app.py:271
msgid "not available"
msgstr "no disponible"

#: module/web/pyload_app.py:509
msgid "Run pyLoadCore.py -s to access the setup."
msgstr "Ejecuta pyLoadCore.py -s para acceder a la instalación."

#: module/network/HTTPDownload.py:245
#, python-format
msgid "Download chunks failed, fallback to single connection | %s"
msgstr "Descarga por partes fallida, se recurre a conexión única | %s"

#: module/PluginThread.py:183
#, python-format
msgid "Download starts: %s"
msgstr "Iniciando descarga: %s"

#: module/PluginThread.py:189
#, python-format
msgid "Download finished: %s"
msgstr "Descarga finalizada: %s"

#: module/PluginThread.py:194 module/PluginThread.py:365
#, python-format
msgid "Plugin %s is missing a function."
msgstr "Plugin %s no encuentra una función."

#: module/PluginThread.py:202 module/PluginThread.py:265
#: module/PluginThread.py:382
#, python-format
msgid "Download aborted: %s"
msgstr "Descarga abortada: %s"

#: module/PluginThread.py:222
#, python-format
msgid "Download restarted: %(name)s | %(msg)s"
msgstr "Descarga reiniciada: %(name)s | %(msg)s"

#: module/PluginThread.py:231 module/PluginThread.py:373
#, python-format
msgid "Download is offline: %s"
msgstr "Descarga está fuera de servicio: %s"

#: module/PluginThread.py:234
#, python-format
msgid "Download is temporary offline: %s"
msgstr "La descarga está temporalmente fuera de servicio: %s"

#: module/PluginThread.py:237 module/PluginThread.py:304
#, python-format
msgid "Download failed: %(name)s | %(msg)s"
msgstr "Descarga fallida: %(name)s | %(msg)s"

#: module/PluginThread.py:254
msgid "Couldn't connect to host or connection reset, waiting 1 minute and retry."
msgstr "No se pudo conectar al servidor o la conexión fue reiniciada, esperando 1 minuto y reintentando."

#: module/PluginThread.py:290
#, python-format
msgid "Download skipped: %(name)s due to %(plugin)s"
msgstr "Descarga omitida: %(name)s debido a %(plugin)s"

#: module/PluginThread.py:361
#, python-format
msgid "Decrypting starts: %s"
msgstr "El descifrado comienza en: %s"

#: module/PluginThread.py:376 module/PluginThread.py:394
#, python-format
msgid "Decrypting failed: %(name)s | %(msg)s"
msgstr "Descifrado fallido: %(name)s | %(msg)s"

#: module/PluginThread.py:388
#, python-format
msgid "Retrying %s"
msgstr "Reintentando %s"

#: module/PluginThread.py:635
#, python-format
msgid "Info Fetching for %(name)s failed | %(err)s"
msgstr "Obtención de información para %(name)s fallida | %(err)s"

#: module/HookManager.py:90 module/plugins/Hook.py:103
#, python-format
msgid "Error executing hooks: %s"
msgstr "Error ejecutando complementos de plugins: %s"

#: module/HookManager.py:140
#, python-format
msgid "Failed activating %(name)s"
msgstr "Falló la activación de %(name)s"

#: module/HookManager.py:144
#, python-format
msgid "Activated plugins: %s"
msgstr "Plugins activados: %s"

#: module/HookManager.py:145
#, python-format
msgid "Deactivate plugins: %s"
msgstr "Desactivar plugins: %s"

#: module/ThreadManager.py:137
#, python-format
msgid "Reconnect Failed: %s"
msgstr "Reconexión Fallida: %s"

#: module/ThreadManager.py:176
msgid "Reconnect script not found!"
msgstr "¡Script de reconexión no encontrado!"

#: module/ThreadManager.py:182
msgid "Starting reconnect"
msgstr "Iniciando reconexión"

#: module/ThreadManager.py:196
msgid "Failed executing reconnect script!"
msgstr "¡Fallo al ejecutar script de reconexión!"

#: module/ThreadManager.py:208
#, python-format
msgid "Reconnected, new IP: %s"
msgstr "Reconectado, nueva IP: %s"

#: module/ThreadManager.py:288
msgid "Not enough space left on device"
msgstr "No hay suficiente espacio libre en el dispositivo"

#: module/plugins/Account.py:85 module/plugins/Account.py:90
#, python-format
msgid "Could not login with account %(user)s | %(msg)s"
msgstr "No se puede iniciar sesión con la cuenta %(user)s | %(msg)s"

#: module/plugins/Account.py:86
msgid "Wrong Password"
msgstr "Contraseña Incorrecta"

#: module/plugins/Account.py:243
#, python-format
msgid "Your Time %s has wrong format, use: 1:22-3:44"
msgstr "Su hora %s tiene un formato incorrecto, use: 1:22-3:44"

#: module/plugins/Account.py:269
#, python-format
msgid "Account %s has not enough traffic, checking again in 30min"
msgstr "La cuenta %s no tiene tráfico suficiente, se comprobará otra vez en 30min"

#: module/plugins/Account.py:276
#, python-format
msgid "Account %s is expired, checking again in 1h"
msgstr "La cuenta %s ha expirado, comprobando de nuevo en 1 hora"

#: module/plugins/crypter/SerienjunkiesOrg.py:126
msgid "Downloadlimit reached"
msgstr "Límite de descargas alcanzado"

#: module/plugins/PluginManager.py:153
#, python-format
msgid "%s has a invalid pattern."
msgstr "%s tiene un patrón inválido."

#: module/plugins/PluginManager.py:272
#, python-format
msgid "Error importing %(name)s: %(msg)s"
msgstr "Error importando %(name)s: %(msg)s"

#: module/plugins/internal/MultiHoster.py:132
msgid "No Hoster loaded"
msgstr "No se cargó ningún proveedor"

#: module/plugins/accounts/BitshareCom.py:38
msgid "Activate direct Download in your Bitshare Account"
msgstr "Activa descarga directa en tu cuenta de Bitshare"

#: module/plugins/container/LinkList.py:64
msgid "LinkList could not be cleared."
msgstr "LinkList no puede ser limpiada."

#: module/plugins/AccountManager.py:88
msgid "Account settings deleted, due to new config format."
msgstr "Ajustes de cuenta eliminados, debido al nuevo formato de la configuración."

#: module/plugins/hoster/BasePlugin.py:65
msgid "Authorization required (username:password)"
msgstr "Autorización requerida (usuario:contraseña)"

#: module/plugins/hoster/SimplydebridCom.py:23
#: module/plugins/hoster/RealdebridCom.py:40
#: module/plugins/hoster/FreeWayMe.py:39 module/plugins/hoster/ZeveraCom.py:21
#: module/plugins/hoster/UnrestrictLi.py:52
#: module/plugins/hoster/Vipleech4uCom.py:30
#: module/plugins/hoster/Premium4Me.py:27 module/plugins/hoster/FastixRu.py:36
#: module/plugins/hoster/SimplyPremiumCom.py:52
#: module/plugins/hoster/MegaDebridEu.py:46
#: module/plugins/hoster/AlldebridCom.py:38
#: module/plugins/hoster/LinksnappyCom.py:29
#: module/plugins/hoster/DebridItaliaCom.py:39
#: module/plugins/hoster/RPNetBiz.py:28
#: module/plugins/hoster/MultiDebridCom.py:40
#: module/plugins/hoster/ReloadCc.py:26 module/plugins/hoster/OverLoadMe.py:38
#: module/plugins/hoster/RehostTo.py:25
#: module/plugins/hoster/PremiumizeMe.py:24
#: module/plugins/hooks/RPNetBiz.py:45
#, python-format
msgid "Please enter your %s account or deactivate this plugin"
msgstr "Por favor, introduce tu cuenta de %s o desactiva este plugin"

#: module/plugins/hoster/FilesMailRu.py:98
#, python-format
msgid "There was HTML Code in the Downloaded File (%s)...redirect error? The Download will be restarted."
msgstr "Había Código HTML en el archivo Descargado (%s)... ¿error de redireccionamiento? La Descarga se reiniciará."

#: module/plugins/hoster/NetloadIn.py:145
#: module/plugins/hoster/NetloadIn.py:169
msgid "File temporarily not available"
msgstr "Archivo no disponible temporalmente"

#: module/plugins/hoster/NetloadIn.py:182
#, python-format
msgid "Netload: waiting between downloads %d s."
msgstr "Netload: esperando entre descargas %d s."

#: module/plugins/hoster/NetloadIn.py:213
#, python-format
msgid "Netload: waiting for captcha %d s."
msgstr "Netload: esperando captcha %d s."

#: module/plugins/hoster/NetloadIn.py:251
msgid "Downloaded File was empty"
msgstr "El archivo descargado estaba vacío"

#: module/plugins/hoster/MegaDebridEu.py:49
#, python-format
msgid "Impossible to connect to %s"
msgstr "Imposible conectar a %s"

#: module/plugins/hoster/MegaDebridEu.py:89
#, python-format
msgid "Impossible to debrid %s"
msgstr "Imposible convertir a premium %s"

#: module/plugins/hoster/UploadedTo.py:129
msgid "API key invalid"
msgstr "Clave de API inválida"

#: module/plugins/hoster/UploadedTo.py:153
#, python-format
msgid "%s: Not enough traffic left"
msgstr "%s: No hay suficiente tráfico"

#: module/plugins/hoster/UploadedTo.py:156
msgid "Traffic exceeded"
msgstr "Tráfico excedido"

#: module/plugins/hoster/RapidshareCom.py:99
msgid "Rapidshare: Traffic Share (direct download)"
msgstr "Rapidshare: Tráfico compartido (descarga directa)"

#: module/plugins/hoster/RapidshareCom.py:126
#: module/plugins/hoster/RapidshareCom.py:193
msgid "Already downloading from this ip address, waiting 60 seconds"
msgstr "Ya estás descargando desde esta dirección IP, esperando 60 segundos"

#: module/plugins/hoster/RapidshareCom.py:130
msgid "Invalid Auth Code, download will be restarted"
msgstr "Código de autenticación inválido, la descarga se reiniciará"

#: module/plugins/hoster/RapidshareCom.py:198
msgid "RapidShareCom: No free slots"
msgstr "RapidShareCom: No quedan espacios disponibles"

#: module/plugins/hoster/RapidshareCom.py:201
msgid "You need a premium account for this file"
msgstr "Necesitas una cuenta premium para este archivo"

#: module/plugins/hoster/RapidshareCom.py:203
msgid "Filename reported invalid"
msgstr "Nombre de archivo inválido"

#: module/plugins/hoster/FileserveCom.py:100
msgid "Parallel download error, now waiting 60s."
msgstr "Error con descargas en paralelo, esperando 60s."

#: module/plugins/hoster/FileserveCom.py:216
msgid "Not logged in."
msgstr "No has iniciado sesión."

#: module/plugins/hoster/MegaNz.py:56
msgid "Decryption failed"
msgstr "Error de descifrado"

#: module/plugins/hoster/MegaNz.py:106
msgid "No file key provided in the URL"
msgstr "No se proporcionó archivo de clave en la URL"

#: module/plugins/hoster/MegaNz.py:118
msgid "Error code:"
msgstr "Código de error:"

#: module/plugins/Container.py:68
msgid "File not exists."
msgstr "El archivo no existe."

#: module/plugins/hooks/UpdateManager.py:97
msgid "Not able to connect server to get updates"
msgstr "No se pudo conectar con el servidor para obtener actualizaciones"

#: module/plugins/hooks/UpdateManager.py:111
msgid "No pyLoad version available"
msgstr "Ninguna versión de pyload disponible"

#: module/plugins/hooks/UpdateManager.py:118
#, python-format
msgid "***  New pyLoad Version %s available  ***"
msgstr "***  Nueva versión %s de pyLoad disponible  ***"

#: module/plugins/hooks/UpdateManager.py:119
msgid "***  Get it here: https://github.com/pyload/pyload/releases  ***"
msgstr "***  Obtenla de aquí : https://github.com/pyload/pyload/releases  ***"

#: module/plugins/hooks/UpdateManager.py:170
#, python-format
msgid "New version of [%(type)s] %(name)s (v%(oldver)s -> v%(newver)s)"
msgstr "Nueva versión de [%(type)s] %(name)s (v%(oldver)s -> v%(newver)s)"

#: module/plugins/hooks/UpdateManager.py:180
#: module/plugins/hooks/UpdateManager.py:185
#, python-format
msgid "Error when updating plugin %s"
msgstr "Error al actualizar el plugin %s"

#: module/plugins/hooks/UpdateManager.py:185
msgid "Version mismatch"
msgstr "No coincide la versión"

#: module/plugins/hooks/UpdateManager.py:196
#, python-format
msgid "Removed blacklisted plugin: [%(type)s] %(name)s"
msgstr "Eliminado el plugin de la lista negra: [%(type)s] %(name)s"

#: module/plugins/hooks/UpdateManager.py:204
msgid "Plugins updated and reloaded"
msgstr "Plugins actualizados y cargados de nuevo"

#: module/plugins/hooks/UpdateManager.py:206
msgid "*** Plugins have been updated, pyLoad will be restarted now ***"
msgstr "*** Los plugins se han actualizado, pyLoad se reiniciará ahora ***"

#: module/plugins/hooks/UpdateManager.py:211
msgid "No plugin updates available"
msgstr "No hay disponibles actualizaciones de plugins"

#: module/plugins/hooks/IRCInterface.py:82
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s "
msgstr "Descarga finalizada: %(name)s @ %(plugin)s "

#: module/plugins/hooks/IRCInterface.py:95
#, python-format
msgid "New Captcha Request: %s"
msgstr "Nueva Petición de Captcha: %s"

#: module/plugins/hooks/IRCInterface.py:96
#, python-format
msgid "Answer with 'c %s text on the captcha'"
msgstr "Conteste con 'c %s texto al captcha'"

#: module/plugins/hooks/Premium4Me.py:29
msgid "Please add your premium.to account first and restart pyLoad"
msgstr "Por favor, añada primero su cuenta premium.to y reinicie pyLoad"

#: module/plugins/hooks/HotFolder.py:81
#, python-format
msgid "Added %s from HotFolder"
msgstr "Añadido %s desde HotFolder"

#: module/plugins/hooks/ExtractArchive.py:94
#, python-format
msgid "No %s installed"
msgstr "%s no instalado"

#: module/plugins/hooks/ExtractArchive.py:96
#: module/plugins/hooks/ExtractArchive.py:101
#, python-format
msgid "Could not activate %s"
msgstr "No se puede activar %s"

#: module/plugins/hooks/ExtractArchive.py:106
msgid "Activated"
msgstr "Activado"

#: module/plugins/hooks/ExtractArchive.py:108
msgid "No Extract plugins activated"
msgstr "No se ha activado ningún plugin de extracción"

#: module/plugins/hooks/ExtractArchive.py:120
#, python-format
msgid "Package %s queued for later extracting"
msgstr "Paquete %s encolado para su posterior extracción"

#: module/plugins/hooks/ExtractArchive.py:143
#, python-format
msgid "Check package %s"
msgstr "Comprobar paquete %s"

#: module/plugins/hooks/ExtractArchive.py:184
#, python-format
msgid "Extract to %s"
msgstr "Extraer a %s"

#: module/plugins/hooks/ExtractArchive.py:199
msgid "No files found to extract"
msgstr "No se encontraron archivos para extraer"

#: module/plugins/hooks/ExtractArchive.py:206
msgid "extracting"
msgstr "extrayendo"

#: module/plugins/hooks/ExtractArchive.py:217
msgid "Password protected"
msgstr "Protegido con contraseña"

#: module/plugins/hooks/ExtractArchive.py:238
msgid "Wrong password"
msgstr "Contraseña errónea"

#: module/plugins/hooks/ExtractArchive.py:246
#, python-format
msgid "Deleting %s files"
msgstr "Borrando archivos %s"

#: module/plugins/hooks/ExtractArchive.py:253
msgid "Extracting finished"
msgstr "Extracción finalizada"

#: module/plugins/hooks/ExtractArchive.py:259
msgid "Archive Error"
msgstr "Error de Archivo"

#: module/plugins/hooks/ExtractArchive.py:261
msgid "CRC Mismatch"
msgstr "El CRC no coincide"

#: module/plugins/hooks/ExtractArchive.py:265
msgid "Unknown Error"
msgstr "Error Desconocido"

#: module/plugins/hooks/ExtractArchive.py:317
msgid "Setting User and Group failed"
msgstr "Error de configuración de usuario y grupo"

#: module/plugins/hooks/ClickAndLoad.py:75
msgid "Click'N'Load: Port 9666 already in use"
msgstr "Click'N'Load: Puerto 9666 ya está en uso"

#: module/plugins/hooks/CaptchaTrader.py:70
#: module/plugins/hooks/Captcha9kw.py:60
#: module/plugins/hooks/ExpertDecoders.py:51
#, python-format
msgid "%s credits left"
msgstr "%s créditos restantes"

#: module/plugins/hooks/CaptchaTrader.py:118
msgid "Could not send response."
msgstr "No se ha podido enviar la respuesta."

#: module/plugins/hooks/CaptchaTrader.py:136
msgid "Your CaptchaTrader Account has not enough credits"
msgstr "Su cuenta de CaptchaTrader no tiene créditos suficientes"

#: module/plugins/hooks/LinkdecrypterCom.py:45
msgid "Crypter list not found"
msgstr "No se ha encontrado lista de cifradores"

#: module/plugins/hooks/LinkdecrypterCom.py:59
msgid "Crypter list is empty"
msgstr "La lista de cifradores está vacía"

#: module/plugins/hooks/XMPPInterface.py:91
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s"
msgstr "Descarga finalizada: %(name)s @ %(plugin)s"

#: module/plugins/hooks/Captcha9kw.py:94
#, python-format
msgid "New CaptchaID from upload: %s : %s"
msgstr "Nuevo CaptchaID desde la carga: %s: %s"

#: module/plugins/hooks/Captcha9kw.py:130
msgid "Your Captcha 9kw.eu Account has not enough credits"
msgstr "Su Cuenta Captcha 9kw.eu no tiene suficientes créditos"

#: module/plugins/hooks/ExternalScripts.py:54
#, python-format
msgid "Installed scripts for %s: "
msgstr "Scripts instalados para %s: "

#: module/plugins/hooks/ExternalScripts.py:69
msgid "Script not executable:"
msgstr "Script no ejecutable:"

#: module/plugins/hooks/ExternalScripts.py:80
#, python-format
msgid "Error in %(script)s: %(error)s"
msgstr "Error en %(script)s: %(error)s"

#: module/plugins/hooks/ExpertDecoders.py:96
msgid "Your ExpertDecoders Account has not enough credits"
msgstr "Su cuenta de  ExpertDecoders no tiene suficientes créditos"

#: module/plugins/hooks/RehostTo.py:32
msgid "Please add your rehost.to account first and restart pyLoad"
msgstr "Por favor, añada primero su cuenta rehost.to y reinicie pyLoad"

#: module/plugins/hooks/PremiumizeMe.py:50
msgid "Please add a valid premiumize.me account first and restart pyLoad."
msgstr "Por favor, agrega primero una cuenta válida de premiumize.me y reinicia pyLoad."

#: module/plugins/hooks/CaptchaBrotherhood.py:70
#, python-format
msgid "%d credits left"
msgstr "%d créditos restantes"

#: module/plugins/Plugin.py:389
msgid "Pil and tesseract not installed and no Client connected for captcha decrypting"
msgstr "Pil y tesseract no están instalados y no hay cliente conectado para descifrar captcha"

#: module/plugins/Plugin.py:393
msgid "No captcha result obtained in appropiate time by any of the plugins."
msgstr "No se obtuvieron resultados para el captcha en el tiempo asignado de ninguno de los plugins."

#: module/plugins/Plugin.py:498 module/plugins/Plugin.py:532
#, python-format
msgid "Setting User and Group failed: %s"
msgstr "Configuración de usuario y grupo fallida: %s"

#: module/CaptchaManager.py:78
msgid "No Client connected for captcha decrypting"
msgstr "No hay cliente conectado para descifrar el captcha"

#: module/Api.py:330
#, python-format
msgid "Added package %(name)s containing %(count)d links"
msgstr "Paquete %(name)s agregado que contiene %(count)d enlaces"

#: module/Api.py:593
#, python-format
msgid "Added %(count)d links to package #%(package)d "
msgstr "Agregados %(count)d enlaces al paquete #%(package)d "

#: module/common/JsEngine.py:156
msgid "No js engine detected, please install either Spidermonkey, ossp-js, pyv8 or rhino"
msgstr "No se detectó ningún motor de js, por favor instale Spidermonkey, ossp-js, pyv8 o rhino"

